### ISSUE TITLE
TransactionSign Error - MetaMask - RPC Error: Expected array params with single object.

### ISSUE THREAD

**Issue by Schwartz10** [ChainSafe/filsnap#89]
When trying to sign transactions with both `transactionSign` and `transactionSignRaw`, MetaMask RPC throws an error:

```
code: -32603
Error: Expected array params with single object.
```

I can reproduce this in [this demo](https://github.com/ChainSafe/filsnap/pull/88) and locally.

**Comment by mpetrunic** [comment]
I'm confused. You are supposed to call `fil_signMessage` and not transactionSign. Could you elaborate further?

**Comment by Schwartz10** [comment]
Ahh! @mpetrunic sorry i mistyped - I meant:

`signMessage` and `signMessageRaw` from https://github.com/ChainSafe/filsnap/wiki/API-Documentation

**Comment by Schwartz10** [comment]
@mpetrunic this is fixed now by #88
